templates/summit/index.html

```diff
@@ -93,8 +96,12 @@ <h2>Featured Speakers</h2>
       <div class="col-2 p-card">
         <div class="p-card__content">
           <div class="p-card__image">
-            <img src="https://events.canonical.com/user/1666/picture-895558193"
-                 alt="Mark Jackels" />
+            {{ image(url="https://events.canonical.com/user/1666/picture-895558193",
```

Comment:
Is this safe? Should we upload these images to the asset server?

Reply:
For now let's keep it this way. We manage the images etc stored on Indico, so I don't think it's a problem.